src/components/DropdownButton/DropdownButton.tsx

```diff
@@ -0,0 +1,35 @@
+import clsx from 'clsx';
+import React, { ReactNode, forwardRef } from 'react';
+import styles from './DropdownButton.module.css';
+import Icon from '../Icon';
+
+type Props = {
+  className?: string;
+  text?: ReactNode;
+};
+
+/**
+ * ```ts
+ * import {DropdownButton} from "@chanzuckerberg/eds-components";
+ * ```
+ *
+ * A styled button with an expand icon to be used in triggering Popovers, Dropdowns, etc.
+ */
+export const DropdownButton = forwardRef<HTMLButtonElement, Props>(
+  ({ className, text, ...rest }, ref) => {
+    return (
+      <button
+        className={clsx(styles['dropdown-button'], className)}
```

Comment:
Do we want to pull out the `clsx` logic into a `componentClassName` variable every time, for consistency?

Reply:
Ya will do!